recipes/lsst-sphgeom/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "lsst-sphgeom" %}
+{% set version = "26.2024.100" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lsst-sphgeom-{{ version }}.tar.gz
+  sha256: 2d25afa6687820c76704232b60f1366dae78899e1549f111e4fd82ad62f8a974
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  script_env: PYTHONUTF8=1
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - lsst-versions >=1.3.0
+    - wheel
+    - pybind11 >=2.5.0
+    - numpy >=1.18
+    - pip
+  run:
+    - python
+    - hpgeom >=0.8.0
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - lsst.sphgeom
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/lsst/sphgeom
+  summary: A spherical geometry library.
+  license: GPL-3.0-only
```

Comment:
I need to fix the sphgeom distribution. It's meant to be dual licensed (since it is used by daf_butler which is dual licensed).

Reply:
Thanks @timj, I fixed it in the recipe!